lineman/app/components/thread_page/activity_card/motion_edited_item_controller.coffee

```diff
@@ -0,0 +1,11 @@
+angular.module('loomioApp').controller 'MotionEditedItemController', ($scope) ->
+  $scope.proposal = $scope.event.version().proposal()
+  version = $scope.event.version()
+  $scope.proposalName =
+    if version.attributeEdited('name')
+      version.changes.name[1]
```

Comment:
Hmm.. I assume we do this because the object changes become serialized out as `["old_value", "new_value"]`?

I wonder if we actually need the old value in there, and if it might be better to just get the new value in the serializer, like

```
object.object_changes.inject({}) do |result, (key, value) |
  result[key] = value[1]
  result
end
```

(erg, maybe not... why are simple hash transformations so funky?)